tests/lwe_to_polynomial/test_rlwe_sk_encrypt.mlir

```diff
@@ -0,0 +1,36 @@
+// RUN: heir-opt %s --lwe-to-polynomial | FileCheck %s
+
+#encoding = #lwe.polynomial_coefficient_encoding<cleartext_start=15, cleartext_bitwidth=15>
+#my_poly = #polynomial.int_polynomial<1 + x**1024>
+#ring = #polynomial.ring<coefficientType = i32, coefficientModulus = 7917 : i32, polynomialModulus=#my_poly>
+#rlwe_params = #lwe.rlwe_params<dimension=2, ring=#ring>
+!plaintext_rlwe = !lwe.rlwe_plaintext<encoding = #encoding, ring = #ring, underlying_type=i3>
+!ciphertext_rlwe = !lwe.rlwe_ciphertext<encoding = #encoding, rlwe_params = #rlwe_params, underlying_type=i3>
+!rlwe_key = !lwe.rlwe_secret_key<rlwe_params=#rlwe_params>
+
+func.func @test_rlwe_encrypt(%arg0: !plaintext_rlwe, %arg1: !rlwe_key) -> !ciphertext_rlwe {
```

Comment:
Apologies, I wrote something confusing. I meant to say that this func.func here should be renamed to test_rlwe_sk_encrypt, not the file name itself (and the same for the pk encrypt test). This is just so that it's easier to disambiguate the two if there are UIs that run through all of these tests and one of them fails.

Reply:
ahhhh got it my bad

Reply:
ok should be supported now